packages/analytics-js/src/components/configManager/types.ts

```diff
@@ -81,6 +85,13 @@ export type SourceConfigResponse = {
     createdAt: string;
     createdBy: string;
     deleted: boolean;
-    dataplanes?: Record<ResidencyServerRegion, RegionDetails>;
+    dataplanes?: Record<ResidencyServerRegion, RegionDetails[]>;
   };
 };
+
+export interface IConfigManager {
```

Comment:
please add all methods that the ConfigManager Class has in this inteface

Reply:
Done